docs/deployment/resource-providers/standalone/docker.md

```diff
@@ -442,7 +270,7 @@ The next chapters show examples of configuration files to run Flink.
     docker exec -t -i "${JM_CONTAINER}" flink run -d -c ${JOB_CLASS_NAME} /job.jar
     ```
 
-### Session Cluster with Docker Compose
+#### Session Cluster with Docker Compose
```

Comment:
Can't we just have a code block for each of the yaml configurations? Not sure whether the section are necessary here.

Reply:
FYI: What about using collapsible blocks to hide long configuration files:
```
<p>
  <a class="btn" data-toggle="collapse" href="#session-cluster-yml" role="button" aria-expanded="false" aria-controls="session-cluster-yml">
    docker-compose.yml
  </a>
</p>
<div class="collapse" id="session-cluster-yml">
  {% highlight yaml %}
version: "2.2"
services:
  jobmanager:
    image: flink:{% if site.is_stable %}{{site.version}}-scala{{site.scala_version_suffix}}{% else %}latest{% endif %}
    ports:
      - "8081:8081"
    command: jobmanager
    environment:
      - |
        FLINK_PROPERTIES=
        jobmanager.rpc.address: jobmanager

  taskmanager:
    image: flink:{% if site.is_stable %}{{site.version}}-scala{{site.scala_version_suffix}}{% else %}latest{% endif %}
    depends_on:
      - jobmanager
    command: taskmanager
    scale: 1
    environment:
      - |
        FLINK_PROPERTIES=
        jobmanager.rpc.address: jobmanager
        taskmanager.numberOfTaskSlots: 2
  {% endhighlight %}
</div>
```
I got inspired by [this blog post](https://www.iditect.com/how-to/52576626.html).
```

Reply:
I'll try it out!